frontend/src/components/table/Table.tsx

```diff
@@ -1,48 +1,52 @@
-import { TableBody } from './TableBody';
-import { TableFooter } from './TableFooter';
-import { TableHeader } from './TableHeader';
-import { Loading } from '../ui/Loading';
-import type { SearchParams } from '@/common/interface';
-import type { TableData } from './interface';
+import { Loading } from '../ui';
+import type { FieldInterface } from '../form';
+import { Toggle } from '../form/components/Toggle';
+import type { Column } from '.';
+import { TableFooter, TableBody, TableHeader } from '.';
+// s
+import useTable from '@/hooks/useTable';
 
 export const Table = ({
-  pageRange,
-  tableData,
-  searchParams,
-  totalRows,
-  handleParamsChange,
+  title,
+  subtitle,
+  // fields,
+  columns,
+  toggle,
 }: {
-  pageRange: number[];
-  tableData: TableData;
-  searchParams: URLSearchParams;
-  totalRows: number;
-  handleParamsChange: (params: SearchParams) => void;
+  title: string;
+  subtitle: string;
+  // fields: FieldInterface[];
```

Comment:
Keeping this?

Reply:
yes, it'll be uncommented on third PR :)